src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -1,62 +1,39 @@
 package org.broadinstitute.dsde.workbench.leonardo.api
 
-import java.time.Instant
-import java.util.UUID
-
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.http.scaladsl.model._
-import akka.http.scaladsl.model.headers.{Cookie, RawHeader}
+import akka.http.scaladsl.model.headers._
 import akka.http.scaladsl.model.ws.{TextMessage, WebSocketRequest}
 import akka.http.scaladsl.testkit.ScalatestRouteTest
 import akka.http.scaladsl.Http
 import akka.stream.scaladsl.{Keep, Sink, Source}
-import org.broadinstitute.dsde.workbench.google.gcs.GcsBucketName
 import org.broadinstitute.dsde.workbench.leonardo.db.TestComponent
 import org.broadinstitute.dsde.workbench.leonardo.GcsPathUtils
-import org.broadinstitute.dsde.workbench.leonardo.model._
 import org.broadinstitute.dsde.workbench.leonardo.service.TestProxy
 import org.broadinstitute.dsde.workbench.leonardo.service.TestProxy.Data
 import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
-import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.time.{Seconds, Span}
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfter, BeforeAndAfterAll, FlatSpec, Matchers}
 
 import scala.collection.immutable
 
 /**
   * Created by rtitle on 8/10/17.
   */
-class ProxyRoutesSpec extends FlatSpec with Matchers with BeforeAndAfterAll with ScalatestRouteTest with ScalaFutures with TestLeoRoutes with TestProxy with TestComponent with GcsPathUtils {
+class ProxyRoutesSpec extends FlatSpec with Matchers with BeforeAndAfterAll with BeforeAndAfter with ScalatestRouteTest with ScalaFutures with TestLeoRoutes with TestProxy with TestComponent with GcsPathUtils {
   implicit val patience = PatienceConfig(timeout = scaled(Span(10, Seconds)))
 
   val clusterName = "test"
   val googleProject = "dsp-leo-test"
-  val TokenCookie = Cookie("FCtoken", "me")
-  val unauthorizedTokenCookie = Cookie("FCtoken", "unauthorized")
-  val expiredTokenCookie = Cookie("FCtoken", "expired")
+  val tokenCookie = HttpCookiePair("FCtoken", "me")
+  val unauthorizedTokenCookie = HttpCookiePair("FCtoken", "unauthorized")
+  val expiredTokenCookie = HttpCookiePair("FCtoken", "expired")
   val serviceAccountEmail = WorkbenchEmail("pet-1234567890@test-project.iam.gserviceaccount.com")
   val userEmail = WorkbenchEmail("user1@example.com")
 
   val routeTest = this
 
-  val c1 = Cluster(
```

Comment:
`ProxyRoutes` doesn't talk to the database, so persisting a cluster is not needed.